app/assets/javascripts/discovery-two-pane.js

```diff
@@ -0,0 +1,7 @@
+$(document).on('click', ".table-two-pane td[class$='display-two-pane'] a", function(e) {
```

Comment:
if this is going to be merged before the change in core then the js needs to be added to the precompiled assets like: https://github.com/theforeman/foreman_discovery/pull/118/files#diff-9c896d7540d8746964738086b33782b7R25